bin/server.js

```diff
@@ -31,6 +32,12 @@ const argv = optimist
         default: 10,
         describe: 'maximum number of tcp sockets each client is allowed to establish at one time (the tunnels)'
     })
+    .options('certKeyPath', {
+        describe: '',
```

Comment:
what about the actual description?

Reply:
Well spotted, thanks! The description is added.